server/data/departureClient.ts

```diff
@@ -9,11 +9,7 @@ export default class DepartureClient {
     this.restClient = new RestClient('departureClient', config.apis.approvedPremises as ApiConfig, token)
   }
 
-  async create(
-    premisesId: string,
-    bookingId: string,
-    departure: Omit<Departure, 'id' | 'bookingId'>,
-  ): Promise<Departure> {
+  async create(premisesId: string, bookingId: string, departure: DepartureDto): Promise<Departure> {
```

Comment:
Confused as to why a departure is of type `DepartureDto` and not `Departure` but returns a `Departure` from the function?

Reply:
So, `DepartureDto` is a `Departure` without the ID and BookingID, and with `moveOnCategory`,  `destinationProvider` and `destinationAp` as UUID strings. When we POST to the API, the Departure is created and returned as a "proper" `Departure` object with the ID and BookingID, as well as the `moveOnCategory`,  `destinationProvider` and `destinationAp` as objects.